README.md

```diff
@@ -262,7 +262,7 @@ Default scheme:
 |Key (key code) | Description|
 |-|-|
 |`Escape` (27) | If the suggestions list is shown - hide it. Defined by `hideList` property. |
-|`ArrowDown` (40) | If the suggestions list is hidden - show it.  Defined by `selectionDown` property. |
+|`ArrowDown` (40) | If the suggestions list is hidden - show it.  Defined by `showList` property. |
```

Comment:
I guess, both properties are correct (selectionDown and showList). So why not leave both?

Reply:
`selectionDown` prop responsible for cycle suggestions is described on the next  line [266](https://github.com/KazanExpress/vue-simple-suggest/pull/175/files/bc8ecc40f7f0e7db24a21700a0f242798cf06708#diff-04c6e90faac2675aa89e2176d2eec7d8L266)
Here I replaced `selectionDown` with `showList`, because it's no more responsible for list opening.

Reply:
Got it.